src/client_scripts/.eslintrc

```diff
@@ -0,0 +1,115 @@
+{
+    "env": {
+      "es6": true,
+      "browser": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "sourceType": "module",
+      "ecmaVersion": "2018"
+    },
+    "extends": [
+      "eslint:recommended",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:testcafe/recommended"
+    ],
+    "plugins": [
+      "@typescript-eslint",
+      "testcafe"
+    ],
+    "rules": {
```

Comment:
You don't have to repeat all rules that are declared in `.eslintrc`'s located in parent directories. Just change the parts that need to be changed.